tests/ga4gh/trs/test_server.py

```diff
@@ -85,3 +87,48 @@ def test_addTool():
     with app.test_request_context(json=MOCK_REQUEST_DATA_1):
         res = addTool.__wrapped__()
         assert isinstance(res, str)
+
+
+def test_toolsIdGet():
+    """Test for getting tool object using `tool_id`.
```

Comment:
I've noticed some things about your docstrings:
* Sometimes you leave a space before and/or after the actual text (not in this case, but I've changed this for some other docstrings). Please don't do that, because it will end up looking weird on the generated API docs.
* Your use of line wraps is rather erratic, because sometimes you wrap lines before the recommended line length of 79 characters. Please always wrap lines just before the word that would exceed the line length above 79 characters.
* For docstrings, [PEP8](https://www.python.org/dev/peps/pep-0008/) recommends the following:
  * Note that most importantly, the `"""` that ends a _multiline_ docstring should be on a line by itself (as you generally do, so that's good)
  * For one liner docstrings, please keep the closing """ on the same line (which you don't do here, because altogether, the docstring including opening and closing `"""` would not exceed 79 characters)

Now, looking through PEP8, I have _just_ read (I never knew that!) that docstrings and comments should actually be limited to 72 characters, whereas all the rest is limited to 79 (which I thought applies to _all_ lines, including docstrings and comments). That's good to know for future projects, but I think since we have used 79 for all lines (including comments and docstrings), we should stick with it now for consistency. For future projects/repos, we can try to keep with 72 for comments and docstrings, as per PEP8.